src/blocks/google-map.v1.yaml

```diff
@@ -0,0 +1,29 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Google map
+type: object
+allOf:
+  - $ref: ../misc/caption.v1.yaml
+  - properties:
+        type:
+            type: string
+            enum:
+              - google-map
+        id:
+            title: Google map ID
+            type: string
+        width:
+            title: Width
+            type: integer
+            minimum: 1
+        height:
+            title: Height
+            type: integer
+            minimum: 1
+        allowFullscreen:
+            type: boolean
+            default: true
+    required:
+      - type
+      - id
+      - width
+      - height
```

Comment:
Could have a default? (The ratio changing shouldn't make a difference with this type of embed.)

Reply:
> Could have a default? (The ratio changing shouldn't make a difference with this type of embed.)

I believe that you're saying that we can expect that all google maps we wish to include will have the same aspect ratio so there is no need to calculate it? This would make the Journal CMS simpler also and I can drop the parameters here.